### DISCUSSION TITLE
Model-generated title and query language mismatch

### DISCUSSION THREAD

**Discussion by ckappgit** [open-webui/open-webui#3670] [General]
**Is your feature request related to a problem? Please describe.**
After executing a specific query, the model generates an automatic title to summarize the results. However, this title is not always accurately reflecting coming from the correct linguistic context. For example, when querying in Chinese, the generated title might be in English or another non-Chinese language.

**Describe the solution you'd like**
Through my experimentation and trials, I found that modifying the prompt words within the query can effectively adjust the output title from the model. The adjusted prompt is as follows:
```
Here is the query:
{{prompt:middletruncate:8000}}

Create a concise, 3-5 word phrase with an emoji as a title for the previous query. Suitable Emojis for the summary can be used to enhance understanding but avoid quotation marks or special formatting. RESPOND ONLY WITH THE TITLE TEXT.ANSWER USING THE SAME LANGUAGE AS THE QUERY.

Examples of titles:
📉 Stock Market Trends
🍪 완벽한 초콜릿 칩 레시피
流媒体音乐的演变
Советы по повышению производительности удаленной работы
Künstliche Intelligenz im Gesundheitswesen
🎮 ビデオゲーム開発の洞察
```

**Additional context**
After my test, it works well on the gemma2 model

**Comment by ckappgit** [comment]
I intended to submit a pull request but couldn't locate the specific source code.

**Comment by tjbck** [comment]
Related #1692

**Comment by timbosssds** [comment]
Any further updated on this one?
Using Gemma2 and english questions are being saved with titles like this 流媒体音乐的演变
